plugins/terminal-proxy/lib/terminalProxy.js

```diff
@@ -391,14 +391,14 @@ TerminalWebsocketProxy.prototype.handleTerminalClientMessage = function(message,
 };
 
 TerminalWebsocketProxy.prototype.netSend = function(buffer) {
-  this.logger.debug(this.identifierString()+' Writing to host socket. Length='+buffer.length);
-  this.logger.log(this.logger.FINER,this.identifierString()+' Content to be sent to host socket=\n'+buffer);
+  this.logger.debug("ZWED0275I", this.identifierString(), buffer.length); //this.logger.debug(this.identifierString()+' Writing to host socket. Length='+buffer.length);
+  this.logger.debug("ZWED0136I", this.identifierString(), buffer); //this.logger.log(this.logger.FINER,this.identifierString()+' Content to be sent to host socket=\n'+buffer);
```

Comment:
Same concern as above... you really do not want to print this by accident, it can be a lot of stuff. best to use logger.trace

Reply:
Changed to trace for the data.